src/Raven.Server/Smuggler/Documents/StreamDestination.cs

```diff
@@ -885,6 +893,19 @@ private void WritePostgreSqlConfiguration(PostgreSqlConfiguration postgreSqlConf
                 _context.Write(_writer, postgreSqlConfig.ToJson());
             }
 
+            private void WriteShardingConfiguration(ShardingConfiguration shardingConfiguration)
+            {
+                _writer.WriteStartObject();
+
+                _writer.WritePropertyName(nameof(shardingConfiguration.Shards));
```

Comment:
anything else we need to persist from here?

Reply:
`Sharding.Prefix` probably, when we'll have it
`Orchestrator` topology ? `BucketMigrations` ? @karmeli87